src/types.ts

```diff
@@ -61,6 +61,15 @@ declare namespace Clova {
     };
   };
 
+  export type EventRequest = {
+    type: 'EventRequest';
+    requestId: string;
+    timestamp: string;
+    locale: string;
+    extensionId: string;
```

Comment:
`locale` and `extensionId` is not on the CEK API document [1], could you confirm this is also showing on the request?

[1] https://clova-developers.line.me/guide/#/CEK/References/CEK_API.md#CustomExtEventRequest

Reply:
Thank you for your comment.

> could you confirm this is also showing on the request?

Yes, it was not in the document. But, I could confirm it.

## user request sample message

```json
{
  "body": {
    "version": "1.0",
    "session": null,
    "context": {},
    "request": {
      "type": "EventRequest",
      "requestId": "8c7f4456-3073-4f6a-b6a4-ede6a700fd64",
      "timestamp": "2018-08-21T00:00:00Z",
      "locale": "ja-JP",
      "extensionId": "xxxxx",
      "intent": { "intent": "", "name": "", "slots": null },
      "event": {
        "namespace": "ClovaSkill",
        "name": "SkillEnabled",
        "payload": null
      }
    }
  }
}
```